<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Mostrar Grafo</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="ID_lineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Origen en x:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="OrigenX_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Origen en y:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="OrigenY_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Destino x:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="DestinoX_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Destino y:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="DestinoY_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Velocidad:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="Velocidad_lineEdit"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Color (rgb)</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Red:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="Red_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Green:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QSpinBox" name="Green_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Blue: </string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSpinBox" name="Blue_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QPushButton" name="Agregar_Inicio_pushButton">
             <property name="text">
              <string>Agregar Inicio</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QPushButton" name="Agregar_Final_pushButton">
             <property name="text">
              <string>Agregar Final</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="QPushButton" name="Mostar_pushButton">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0" colspan="2">
            <widget class="QPushButton" name="Ordenar_ID_pushButton">
             <property name="text">
              <string>Ordenar por ID</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0" colspan="2">
            <widget class="QPushButton" name="Ordenar_Distancia_pushButton">
             <property name="text">
              <string>Ordenar por Distancia</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0" colspan="2">
            <widget class="QPushButton" name="Ordenar_Velocidad_pushButton">
             <property name="text">
              <string>Ordenar por Velocidad</string>
             </property>
            </widget>
           </item>
           <item row="16" column="0" colspan="2">
            <widget class="QPushButton" name="Mostrar_Grafo">
             <property name="text">
              <string>Mostrar Grafo</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="salida"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="2">
         <widget class="QPushButton" name="ordenarVelocidadT_pushButton">
          <property name="text">
           <string>Ordenar por Velocidad</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="ordenarDistanciaT_pushButton">
          <property name="text">
           <string>Ordenar por Distancia</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="table"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="ordenarIDT_pushButton">
          <property name="text">
           <string>Ordenar por ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="mostar_tabla_pushButton">
          <property name="text">
           <string>Mostar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>PÃ¡gina</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Dibujar">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
